packages/docs/src/scripts/components/UsaBanner.jsx

```diff
@@ -12,7 +12,7 @@ const UsaBanner = function(props) {
         <div className="ds-u-display--flex ds-u-flex-direction--row ds-u-align-items--start ds-u-sm-align-items--center">
           <img
             className="c-usa-banner__header-flag"
-            src="/public/images/us_flag_small.png"
+            src="./public/images/us_flag_small.png"
             alt="U.S. flag"
           />
           <p className="c-usa-banner__header-text">
```

Comment:
I think you might need to add the dot to relative paths on the images on line 40 and 52 as well